azurerm/internal/services/netapp/resource_arm_netapp_volume.go

```diff
@@ -226,6 +230,7 @@ func resourceArmNetAppVolumeCreateUpdate(d *schema.ResourceData, meta interface{
 			UsageThreshold: utils.Int64(storageQuotaInGB),
 			ExportPolicy:   expandArmNetAppVolumeExportPolicyRule(exportPolicyRule),
 		},
+		Tags: tags.Expand(t),
```

Comment:
_minor_ no need to change it but this is preferred
```suggestion
		Tags: tags.Expand(d.Get("tags").(map[string]interface{})),
```